### ISSUE TITLE
Compilation problem in ubuntu bionic

### ISSUE THREAD

**Issue by luispabon** [cloudflare/boringtun#97]
While trying to create a docker image using cargo install, compilation failed at

```
error[E0308]: mismatched types
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/boringtun-0.2.0/src/device/tun_linux.rs:77:29
   |
77 |                 ifru_flags: IFF_TUN | IFF_NO_PI | IFF_MULTI_QUEUE,
   |                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected i16, found i32
```
Ubuntu bionic
cargo 1.35.0
rustc 1.35.0

**Comment by vkrasnov** [comment]
This is fixed in the master branch

**Comment by keith24** [comment]
I got this using `cargo install`.  It _did_ work for me to compile from source: 

```sh
$ git clone https://github.com/cloudflare/boringtun.git 
$ cd boringtun
$ cargo build --bin boringtun --release
```

There is a duplicate issue #108 with [a workaround](https://github.com/cloudflare/boringtun/issues/108#issuecomment-553453465): 

> Can you try replacing this with: `(IFF_TUN | IFF_NO_PI | IFF_MULTI_QUEUE) as _,` ?

**Comment by vkrasnov** [comment]
Fixed in 0.3.0 release
